src/pybind/mgr/cephadm/inventory.py

```diff
@@ -2036,8 +2036,8 @@ def save_cert(self, entity: str, cert: str, service_name: str = '', host: str =
             var = service_name if entity in self.service_name_cert else host
             j = {}
             self.known_certs[entity][var] = cert_obj
-            for service_name in self.known_certs[entity].keys():
-                j[var] = Cert.to_json(self.known_certs[entity][var])
+            for k in self.known_certs[entity]:
+                j[k] = Cert.to_json(self.known_certs[entity][k])
```

Comment:
niptick: I can never remember the exact times the static checking tools reject single letter variables, but even if the tool is OK with these, I think that it'd still be good to avoid adding new ones. In short, how about `key` or `cert_key` instead of just `k`?

Reply:
I updated the code, I think `cert_key ` is for sure more readable :+1: